identify/extensions.py

```diff
@@ -304,6 +306,8 @@
     'go.mod': {'text', 'go-mod'},
     'go.sum': {'text', 'go-sum'},
     'Jenkinsfile': EXTENSIONS['jenkins'],
+    'Justfile': EXTENSIONS['just'],
```

Comment:
`just` appears to be case insensitive -- I don't think identify should support `Justfile` given the documented examples are all `justfile` or `.justfile`

Reply:
> `just` appears to be case insensitive -- I don't think identify should support `Justfile` given the documented examples are all `justfile` or `.justfile`

There are other filenames that are case-insensitive, such as README/ReadMe/Readme, LICENSE — I should have noticed that identify only handles one case for them and followed that pattern.

Would you like me to remove that line in another commit, or is that something you would like to handle? (I ask because it seems you don't like multiple commits.)

The other option may be for a bigger change of being able to mark particular names as being case-insensitive. (But not in this PR, obviously.)